### PR TITLE
Eswe 415 create unit test for education api

### PR DESCRIPTION
**Pull request by sacintha**

### CODE REVIEW

**Review by theDustRoom — CHANGES_REQUESTED** [review]
As these are standard endpoint checks used within HMPPS projects we need to ensure that these health and info endpoints are functioning as expected so the tests should be in place to ensure this.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/IntegrationTestBase.kt [hunk 1]
```diff
@@ -26,28 +26,31 @@ import uk.gov.justice.digital.hmpps.educationemploymentapi.helpers.JwtAuthHelper
...
   @Autowired
   lateinit var jwtAuthHelper: JwtAuthHelper
-
   internal fun setAuthorisation(
     user: String = "test-client",
     roles: List<String> = listOf()
+  ): (HttpHeaders) {
+    return jwtAuthHelper.setAuthorisationForUnitTests(user, roles)
+  }
+  /*internal fun setAuthorisation(
```
[the remark this answers is not in the document]
Comment (sacintha) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/IntegrationTestBase.kt]: I have removed the comment

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/health/HealthCheckTest.kt [hunk 2]
```diff
@@ -1,68 +1,62 @@
...
-      .exchange()
-      .expectStatus().isOk
-      .expectBody()
-      .jsonPath("components.healthInfo.details.version").value(
-        Consumer<String> {
-          assertThat(it).startsWith(LocalDateTime.now().format(DateTimeFormatter.ISO_DATE))
-        }
-      )
-  }
+  /* @Test  @Test
```
[the remark this answers is not in the document]
Comment (sacintha) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/health/HealthCheckTest.kt]: I have added them

File: src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/health/InfoTest.kt [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
 import uk.gov.justice.digital.hmpps.educationemploymentapi.integration.IntegrationTestBase
-import java.time.LocalDateTime
-import java.time.format.DateTimeFormatter
 
 class InfoTest : IntegrationTestBase() {
 
-  @Test
+ /* @Test
```
[the remark this answers is not in the document]
Comment (sacintha) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/integration/health/InfoTest.kt]: I have added them

**Pull request merged**